src/main/java/org/apache/maven/plugins/changes/ChangesReport.java

```diff
@@ -75,6 +75,14 @@ public class ChangesReport extends AbstractChangesReport {
     @Parameter(property = "changes.addActionDate", defaultValue = "false")
     private boolean addActionDate;
 
+    /**
+     * Whether the changelog action text should be passed as raw text to the Doxia sink or not.
```

Comment:
"raw text" should be defined here and an example added

also "changelog action" is a term I really had to think about, here and in the code. Is there any more obvious name we can use for that? E.g. change description?

Reply:
i've added an example in the javadoc and changes the text to "change description".